test/commands/provar/automation/project/compile.nut.ts

```diff
@@ -55,22 +70,33 @@ describe('provar automation project compile NUTs', () => {
     expect(res.jsonOutput).to.deep.equal(validateConstants.missingFileJsonError);
   });
 
-  it('Compile command should be successful', () => {
-    execCmd<SfProvarCommandResult>(
-      `${commandConstants.SF_PROVAR_AUTOMATION_CONFIG_GENERATE_COMMAND} -p ${FILE_PATHS.COMPILE_FILE}`
-    );
-    execCmd<SfProvarCommandResult>(
-      `${commandConstants.SF_PROVAR_AUTOMATION_CONFIG_LOAD_COMMAND} -p ${FILE_PATHS.COMPILE_FILE}`
-    );
-    const SET_PROVAR_HOME_VALUE = '"./ProvarHome"';
-    const SET_PROJECT_PATH_VALUE = '"./ProvarRegression/AutomationRevamp"';
-    // set provarHome and projectPath locations
-    execCmd<SfProvarCommandResult>(
-      `${commandConstants.SF_PROVAR_AUTOMATION_CONFIG_SET_COMMAND} "provarHome"=${SET_PROVAR_HOME_VALUE}`
-    );
-    execCmd<SfProvarCommandResult>(
-      `${commandConstants.SF_PROVAR_AUTOMATION_CONFIG_SET_COMMAND} "projectPath"=${SET_PROJECT_PATH_VALUE}`
-    );
+  it('Compile command should be successful', async () => {
+    // Read the config.json file
+    const configFilePatheData = fileSystem.readFileSync(configFilePath, { encoding: 'utf8' });
+
+    // Parse the JSON data
+    const configFilePathParsed = JSON.parse(configFilePatheData);
+    configFilePathParsed['PROVARDX_PROPERTIES_FILE_PATH'] = path.join(process.cwd(), './provardx-properties.json');
+    // Convert the updated JSON object back to a string
+    const updatedCongiFileData = JSON.stringify(configFilePathParsed, null, 4);
+
+    // Write the updated JSON string back to the config.json file
+    fileSystem.writeFileSync(configFilePath, updatedCongiFileData, 'utf8');
+
+    const SET_PROVAR_HOME_VALUE = path.join(process.cwd(), './ProvarHome').replace(/\\/g, '/');
+    const SET_PROJECT_PATH_VALUE = path.join(process.cwd(), './ProvarRegression/AutomationRevamp').replace(/\\/g, '/');
+    interface PropertyFileJsonData {
+      [key: string]: string | boolean | number;
+    }
+    const jsonFilePath = FILE_PATHS.PROVARDX_PROPERTIES_FILE;
```

Comment:
if this variable is needed at multiple places, we can have it as a class variable

Reply:
Done